### DISCUSSION TITLE
What is the correct way to turn off the metrics export ?

### DISCUSSION THREAD

**Discussion by ashutoshraina** [open-telemetry/opentelemetry-java#2591] [Q&A] [answered]
Is there a flag to turn of the metrics export and only have the traces exported ? I looked on the readme but I could not find a property in the OTEL resource attributes.

**Comment by jkwatson** [comment, marked as answer]
If you're asking about the SDK, then just not configuring it to do metrics is enough. 

If you're asking about the auto-instrumentation agent, then I recommend heading over to the discussions in that repository: https://github.com/open-telemetry/opentelemetry-java-instrumentation/discussions

**Reply by ashutoshraina** [reply]
My bad, this was about the agent. I will close this and move the discussion to the agent Repo.
